invokeai/frontend/web/src/services/api/endpoints/workflows.ts

```diff
@@ -0,0 +1,30 @@
+import { logger } from 'app/logging/logger';
+import { Workflow, zWorkflow } from 'features/nodes/types/types';
+import { api } from '..';
+import { paths } from '../schema';
+
+export const workflowsApi = api.injectEndpoints({
+  endpoints: (build) => ({
+    getWorkflow: build.query<Workflow | undefined, string>({
+      query: (workflow_id) => `workflows/i/${workflow_id}`,
```

Comment:
I am curious why you chose to do this instead of an endpoint that takes `image_name`. That would allow us to fetch workflows when we may not have the whole DTO (not sure if that is ever really the case but playing devils advocate)

Reply:
Could certainly add that route, the internal logic is there just no route exposing it. Just did this bc its fewer changes